infra/nodes/akka-node-001.json

```diff
@@ -0,0 +1,32 @@
+{
+    "chef_environment": "_default", 
+    "ipaddress-internal": "10.132.0.7", 
+    "ipaddress": "104.155.50.175", 
+    "name": "akka-node-001", 
+    
+    "akka": {
+      "seed_nodes": "[\"10.132.0.7\"]",
+      
+      "multi_node": [
+        "10.132.0.7",
+        "10.132.0.7",
+        "10.132.0.7",
+        "10.132.0.7",
+        "10.132.0.7",
+        "10.132.0.7",
+        "10.132.0.8"
+      ]
+    },
+    
+    "java": {
+      "jdk_version": 8,
+      "oracle": {
+        "accept_oracle_download_terms": true
+      },
+      "url": "http://download.oracle.com/otn-pub/java/jdk/8u112-b15/server-jre-8u112-linux-x64.tar.gz"
```

Comment:
we install java manually instead from repo to have a locked version as well as the oracle one.
Could be done via isntallers but is more pain IMHO